judge/models/problem.py

```diff
@@ -23,6 +24,12 @@
            'License', 'Solution', 'TranslatedProblemQuerySet', 'TranslatedProblemForeignKeyQuerySet']
 
 
+def disallowed_characters_validator(text):
```

Comment:
Since the infrastructure is set up, please add a few tests for this validator in `tests/problem.py` (https://docs.djangoproject.com/en/2.2/topics/testing/tools/#overriding-settings and https://github.com/DMOJ/online-judge/blob/master/judge/models/tests/test_contest.py#L566 may be useful).